airbyte-integrations/connector-templates/source-python/integration_tests/standard_source_test.py.hbs

```diff
@@ -30,22 +30,42 @@ from base_python_test import StandardSourceTestIface
 
 
 class Source{{properCase name}}StandardTest(StandardSourceTestIface):
-    def __init__(self):
-        pass
+    """
+    If you don't need to reimplement any of the python methods below, you can consider deriving from DefaultStandardSourceTest instead
```

Comment:
i personally wouldn't mention `DefaultStandardSourceTest`. i generally think we want to prevent the developer with fewer choices in this context where they are trying to learn what's happening. this is my own bias though, so ignore if you prefer it as is.

Reply:
Seems like the standard python test are being replaced so this might disappear?